test/exercism/html_formatter_test.rb

```diff
@@ -37,11 +48,11 @@ def get_lexer(language)
   end
 
   def load_sample(name, language)
```

Comment:
`load_sample` should be returning the tokens that you will be using as input to the thing you are testing.
In this case the `lexemes`. The actual call to the thing you are testing should be happening within the test.